app/main.py

```diff
@@ -1,2 +1,98 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+INITIAL_CAPACITY = 8
```

Comment:
These constants and Node class should be defined  inside of a Dictionary since it is a part of its implementation and cannot be used outside it